staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_whitebox_test.go

```diff
@@ -720,6 +728,7 @@ func testCacherSendBookmarkEvents(t *testing.T, allowWatchBookmarks, expectedBoo
 		t.Fatalf("Failed to create watch: %v", err)
 	}
 
+	errorCh := make(chan error, 1)
```

Comment:
defer close this channel?

Reply:
I didn't think we'll need to close this since it's used in a `select` statement, but if you think it's necessary I'll update that.